### PR TITLE
Document -Xverify:none to work around VerifyError

### PR DESCRIPTION
**Pull request by mkitti**
This documents how to work around `java.lang.VerifyError: Expecting a stack map frame` when insert a Java 8 compiled ImageJ1 into FIJI as reported here and on image.sc:
https://github.com/imagej/ij1-patcher/issues/50

Also documentation is added on the use of ImageJ.cfg, replacing prior references to jvm.cfg.

### CODE REVIEW

**Review by ctrueden — CHANGES_REQUESTED** [review]
Thanks, @mkitti! Could you please confirm that the new instructions work as described, on both Windows and non-Windows systems? If you need additional testers for particular platforms, just say so here or in the chat, and others can help as well.

File: _pages/learn/launcher.md [hunk 1]
```diff
@@ -74,11 +74,30 @@ There are basically two ways to do that:
 
 {% include notice icon="note" content="Even if you do not pass ImageJ options at all, you need to add the separator, otherwise ImageJ thinks you passed it an ImageJ option. Example:<br>`./ImageJ-linux64 -XX:+HeapDumpOnOutOfMemoryError --`" %}
 
-- By modifying/creating the file `jvm.cfg` in the same directory as the ImageJ launcher.
+- By modifying/creating the file `ImageJ.cfg` in the same directory as the ImageJ launcher.
```
[the remark this answers is not in the document]
Comment (imagejan) [reply _pages/learn/launcher.md:77]: From https://github.com/imagej/imagej-launcher/blob/a1b5432d6601690faca4e0aa196201e1f69d7f19/src/main/c/config.c#L186-L208, it seems that `ImageJ.cfg` is parsed *if* it exists, and `jvm.cfg` is only used as a fallback, if no `ImageJ.cfg` is present?

[the remark this answers is not in the document]
Comment (ctrueden) [reply _pages/learn/launcher.md:77]: OK, thanks @imagejan. It seems we should change the docs as @mkitti has done here, then.

[the remark this answers is not in the document]
Comment (mkitti) [reply _pages/learn/launcher.md:77]: I will test it shortly just to be sure, but yes, I was going off the source code in this case.

[the remark this answers is not in the document]
Comment (mkitti) [reply _pages/learn/launcher.md:77]: This looks like the originating ticket:
https://imagej.net/tickets/1079.html

**Pull request merged**